install_config/install/deploy_router.adoc

```diff
@@ -90,6 +94,7 @@ users:
 - system:serviceaccount:default:router
 ----
 ====
```

Comment:
The manual editing of the SCC here could be updated to use the newer `add-scc-to-user` command if you wanted, as seen here:

https://docs.openshift.org/latest/admin_guide/manage_scc.html#add-an-scc-to-a-user-or-group